README.md

```diff
@@ -1,4 +1,8 @@
-# tensorflow-yolo-v3
+# tensorflow-yolo-v3 (fork)
```

Comment:
This should not be changed

Reply:
Thanks. Like I said - I haven't done a proper pull request before, so I wasn't sure of the proper etiquette.